<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTagListViewClass</class>
 <widget class="QDialog" name="CTagListViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qrfe/images/qrfe.qrc">
    <normaloff>:/ams logos/mtiWindowIcon</normaloff>:/ams logos/mtiWindowIcon</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../Qrfe/images/qrfe.qrc">:/window images/advancedSettingsImage</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tag List </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Reader:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="readerBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="modeBox">
     <item>
      <property name="text">
       <string>All tags since application start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All tags of last scan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All current tags</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="showButton">
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Tag List:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
 </resources>
 <connections/>
</ui>
